homeassistant/components/air_quality/__init__.py

```diff
@@ -149,6 +149,6 @@ def state(self) -> StateType:
         return self.particulate_matter_2_5
 
     @property
-    def unit_of_measurement(self) -> str:
+    def unit_of_measurement(self) -> str | None:
```

Comment:
I'm a little confused by this one.
Why is this change needed when it explicitely only returns a string below?